plugins/Goals/API.php

```diff
@@ -219,6 +225,11 @@ private function checkName($name)
         return urldecode($name);
     }
 
+    private function checkDescription($description)
+    {
+        return urldecode($description);
```

Comment:
as the description field is VARCHAR 255 maybe we should check here that the description is maximum 255 chars?